composer.json

```diff
@@ -5,7 +5,7 @@
     "description": "Rector upgrades rules for Sylius",
     "require": {
         "php": "^8.0",
-        "rector/rector": "^0.13"
```

Comment:
Is it problematic to support ^0.13 and ^0.14 at the same time?

Reply:
I suppose, that the problem is only with testing as the API of RectorTestCase changed between [^0.14](https://github.com/rectorphp/rector/blob/0.14.8/packages/Testing/PHPUnit/AbstractRectorTestCase.php#L81) and [^0.13](https://github.com/rectorphp/rector/blob/0.13.10/packages/Testing/PHPUnit/AbstractRectorTestCase.php#L91)